src/index.html

```diff
@@ -4,14 +4,236 @@
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0,
+               maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://github.com/Dimoshua/layout_html-form.git"
+      method="post"
+      oninput="result.value=myrange.value"
+    >
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <label for="surname" class="block">
+          Surname:
+          <input
+            type="text"
+            id="surname"
+            name="Personal_information"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label for="name" class="block">
+          Name:
+          <input
+            type="text"
+            id="name"
+            name="Personal_information"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label for="age" class="block">
+          How old are You?
+          <input
+            type="number"
+            id="age"
+            name="Personal_information"
+            min="1"
+            max="100"
+            value="12"
+            required
+          >
+        </label>
+
+        <label for="dateOfBirth" class="block">
+          Full date of birth:
+          <input
+            type="date"
+            id="dateOfBirth"
+            min="01.01.1922"
+            max="01.01.2022"
+            name="Personal_information"
+            required
+          >
+        </label>
+
+        <label for="IofAgree" class="block">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            id="IofAgree"
+            name="Personal_information"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="field">
+        <legend>Registration</legend>
+        <label for ="myemail" class="block">
+          E-mail:
+          <input
+            type="email"
+            id="myemail"
+            name="Registration"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label for="mypassword" class="block">
+          Password:
+          <input
+            type="password"
+            id="mypassword"
+            name="Registration"
+            pattern="\w+"
+            minlength="5"
+            maxlength="32"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="field">
+        <legend>An interesting fact about you!</legend>
+        <div class="block">Do you love cats?
+          <label for="yescats">
+            <input type="radio"
```

Comment:
```suggestion
            <input 
              type="radio"
```
start first property also on new line